Libs/MRML/DisplayableManager/vtkMRMLCameraWidget.cxx

```diff
@@ -610,8 +618,13 @@ bool vtkMRMLCameraWidget::ProcessRotate(vtkMRMLInteractionEventData* eventData)
 
   bool wasCameraNodeModified = this->CameraModifyStart();
 
-  camera->Azimuth(rxf);
-  camera->Elevation(ryf);
+  if (this->stabilize3D) {
```

Comment:
There are axis-preferred rotation modes (when you cannot tilt by more than 90 degrees in a selected axis), which I think would work better than such a rigid locking. Try to load a mesh file in "3D Viewer" built-in Windows application to see how that works.

The current implementation is very simple, though, so it is OK to keep it as is, the improvement is just for considering it in the future.

Reply:
Will check this out.